Gopkg.testing.toml

```diff
@@ -0,0 +1,4 @@
+[[constraint]]
```

Comment:
since this is a library we shouldn't use gopkg

Reply:
It was needed before transactions in `go-git` was in a release. I believe we should add at least `go.mod` for these cases.

I'm deleting `Gopkg.testing.toml` and `dep` calls in `.travis.yml`.